spec/puppet-lint/plugins/check_trailing_comma/check_trailing_comma_spec.rb

```diff
@@ -55,6 +55,17 @@ class { '::apache':
             '/etc/baz.conf', '/etc/baz.conf.d'
              ],
         }
+
+        function myfunc (
+          Mymod::Mytype $arg1,
+          String[1] $arg2,
+        ) >> Mymod::Mytype {
```

Comment:
What's this syntax? I haven't seen this before.

Reply:
Which syntax? The custom data types or typed function return values? They are both new with Puppet 4.x:
https://puppet.com/docs/puppet/5.5/lang_type_aliases.html
and
https://puppet.com/docs/puppet/5.5/lang_write_functions_in_puppet.html#return-types